tests/product/timing_test_decorator.py

```diff
@@ -0,0 +1,65 @@
+import logging
```

Comment:
How does it relate to `https://github.com/prestodb/presto-admin/commit/af804f7384e3ab5af4efe667398fff2e18060c62`? Can we revert that commit?